From a2c655d01a3f70a1787650c634af51efa61dbb2d Mon Sep 17 00:00:00 2001
From: Gyorgy Szing <Gyorgy.Szing@arm.com>
Date: Tue, 8 Feb 2022 17:06:37 +0000
Subject: [PATCH 1/2] Disable using hard-coded attestation key

Modify platform config to disable using a hard-coded attestation
key.

Signed-off-by: Gyorgy Szing <Gyorgy.Szing@arm.com>
---
 api-tests/platform/targets/tgt_dev_apis_linux/nspe/pal_config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/api-tests/platform/targets/tgt_dev_apis_linux/nspe/pal_config.h b/api-tests/platform/targets/tgt_dev_apis_linux/nspe/pal_config.h
index 6112ba7..1cdf581 100755
--- a/api-tests/platform/targets/tgt_dev_apis_linux/nspe/pal_config.h
+++ b/api-tests/platform/targets/tgt_dev_apis_linux/nspe/pal_config.h
@@ -60,7 +60,7 @@ typedef uint32_t            cfg_id_t;
 #define CRYPTO_VERSION_BETA3
 
 /* Use hardcoded public key */
-#define PLATFORM_OVERRIDE_ATTEST_PK
+//#define PLATFORM_OVERRIDE_ATTEST_PK
 
 /*
  * Include of PSA defined Header files
-- 
2.34.1

